test/hack/cleanup/main.go

```diff
@@ -83,7 +84,7 @@ func main() {
 			if err != nil {
 				logger.With("type", resources[i].Type()).Errorf("%v", err)
 			}
-			if err = fireMetric(ctx, cloudWatchClient, fmt.Sprintf("%sDeleted", resources[i].Type()), float64(len(cleaned))); err != nil {
+			if err = fireMetric(ctx, timestreamwriteClient, fmt.Sprintf("%sDeleted", resources[i].Type()), float64(len(cleaned))); err != nil {
```

Comment:
Should we wrap the client with our own methods to make this more idiomatic?

```go
metricsClient.FireMetric(name string, value float64)
```

Reply:
If metricsClient is an interface, we can also swap out the implementation more easily in the future